sources/packages/api/src/route-controllers/institution/institution.controller.ts

```diff
@@ -649,4 +691,17 @@ export class InstitutionController extends BaseController {
       totalUsers: institutionUserAndCount[1],
     };
   }
+  /**
+   * API to get the lookup values for admin role.
+   * @returns Admin Roles.
+   */
+  @IsInstitutionAdmin()
+  @Get("admin-roles")
+  async getAdminRoles(): Promise<UserRoleOptionDTO[]> {
+    const userRoles = await this.institutionService.getAdminRoles();
+    return userRoles.map((role) => ({
+      name: role.role || role.type,
+      code: role.role || role.type,
+    }));
```

Comment:
do name and code have the same attribute values ?

Reply:
Ya. This API is to supply values to drop down component, in user scenarios(before and now) the code and name have the same values. 

This is an example of already existing code for loading non-admin roles. 

![image](https://user-images.githubusercontent.com/54600590/149821464-da5eedd6-cf9d-48d7-b948-3dcd5f4e5737.png)